.github/workflows/main.yml

```
@@ -0,0 +1,22 @@
+name: Github Secret Scanner
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    env:
+      REPO: https://github.com/projecttacoma/fqm-execution
+      REMOTE_EXCLUDES_URL: https://raw.githubusercontent.com/projecttacoma/fqm-execution/master/.github/gitleaks.toml
```

```suggestion
      REPO: https://github.com/projecttacoma/cql-module-generator
      REMOTE_EXCLUDES_URL: https://raw.githubusercontent.com/projecttacoma/cql-module-generator/master/.github/gitleaks.toml
```
Hah! This might be part of the issue. We will also need it to be in `master` so perhaps we will be merging this without gitleaks being happy just to get that config file put into master.